src/components/seeds/database/columns.ts

```diff
@@ -1,266 +1,268 @@
-import strings from 'src/strings';
+import strings, { ILocalizedStrings } from 'src/strings';
 import { DatabaseColumn } from '@terraware/web-components/components/table/types';
 
-const COLUMNS: DatabaseColumn[] = [
+export type LocalizableDatabaseColumn = Omit<DatabaseColumn, 'name'> & { code: keyof ILocalizedStrings };
+
+const COLUMNS: LocalizableDatabaseColumn[] = [
```

Comment:
Optionally,, move this into the component scope so it's evaluated on render time.
Another option, convert this into a function that returns the list of columns, so strings are evaluate when the function is called.

Reply:
Converting to a function seems like a good way to go, yeah. That'll also make it easy to memoize.

Reply:
Updated this to use a function, which definitely seems cleaner. Thanks for that suggestion. It's https://github.com/terraware/terraware-web/pull/1031/commits/6dcd78b75b83c1674fca7ebee0350935c30dcc23 if you want to see it in isolation.

Reply:
This looks good, thanks @sgrimm

Reply:
Updated all the other tables too. Their column lists are all way shorter than the accessions table so it seemed cleaner to inline them into the render functions. (In some cases, they were already inlined and I didn't have to change anything.)